packages/contracts/contracts/optimistic-ethereum/OVM/predeploys/OVM_UpgradeExecutor.sol

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: MIT
+// @unsupported: evm
+pragma solidity >0.5.0 <0.8.0;
+
+/* Library Imports */
+import { Lib_ExecutionManagerWrapper } from "../../libraries/wrappers/Lib_ExecutionManagerWrapper.sol";
+
+/**
+ * @title OVM_UpgradeExecutor
+ * @dev The OVM_UpgradeExecutor is the contract which authenticates and executes (i.e.
```

Comment:
`ChugSplashDeployer` contract would replace this one. I think we can probably remove this?

Reply:
Yeah I think we should remove this contract in favor of later introducing the `ChugSplashDeployer` with tests.